pyaedt/application/Design.py

```diff
@@ -212,7 +212,7 @@ def list_difference(list1, list2):
 class DesignCache(object):
     """DesignCache class.
```

Comment:
```suggestion
    """Provides for analyzing the differences in the state of a design between two points in time.
```

I'm also ok with "Used for..."